# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vips/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vips-7.10.21/libsrcCC/VDisplay.cc.vanilla	2009-04-12 22:27:06.000000000 +0200
+++ vips-7.10.21/libsrcCC/VDisplay.cc	2009-04-12 22:27:18.000000000 +0200
@@ -32,6 +32,7 @@
 #include <vips/intl.h>
 
 #include <stdlib.h>
+#include <string.h>
 
 #include <vips/vips.h>
 #include <vips/vipscpp.h>
